d2core/d2systems/scene_ebiten_splash.go

```diff
@@ -105,7 +107,7 @@ func (s *EbitenSplashScene) createSplash() {
 	size := 10
 
 	totalW, totalH := len(flags[0])*size, len(flags)*size
-	ox, oy := (800-totalW)/2, (600-totalH)/2
+	ox, oy := (800-totalW)/2, (600-totalH)/2 //nolint:gomnd // halving things...
```

Comment:
i suppose i should have just done `>>1` here

Reply:
:shrug: either way, I think the way you did it helps keep it clearer for people unfamiliar with bitwise stuff

Reply:
true. I've started doing `>>1` in other code to satisfy the lint gods without having to write lint directives.